Bottom: 2bb56fabeb048af0f39eddd5c5571d115662f3c3
Top:    74dde53935add978e9732721b575c752515db728
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 19:34:09 +0000

Refresh of more-play-with-quad-edge-data

---

diff --git a/src/hid/common/vertex3d.c b/src/hid/common/vertex3d.c
index 174bad6..c89c231 100644
--- a/src/hid/common/vertex3d.c
+++ b/src/hid/common/vertex3d.c
@@ -1,17 +1,8 @@
 #include <stdlib.h>
 
-static int global_vertex3d_count;
-
-typedef struct
-{
-  float x;
-  float y;
-  float z;
-  int id;
+#include "vertex3d.h"
 
-  /* XXX: STEP specific - breaks encapsulation */
-  int vertex_identifier;
-} vertex3d;
+static int global_vertex3d_count;
 
 vertex3d *
 make_vertex3d (float x, float y, float z)
